vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -0,0 +1,175 @@
+---
+sidebarDepth: 2
+---
+# Manage NGINX
+
+## Verify the NGINX Ingress Install
+We recommend setting up a test application to verify that the ingress is working.
+
+1. Create a simple application from the `Cloud Shell`:
+```
+kubectl create deployment hello-server --image=us-docker.pkg.dev/google-samples/containers/gke/hello-app:1.0
```

Comment:
This is specific to GKE but I don't have a better idea currently without doing some testing. :(

Reply:
@nshaw i kept the language but added a note, in bold, to clarify that step 1. applies to gke only and to start at step 2. if you're using another service. is that direction correct?

Reply:
@Lyd1aCla1r3 Sorry, I wasn't clear. The step is required for any server - you have to deploy a sample service to test the controlller. Let's remove the Note from above and just drop the "from the Cloud Shell" comment on Step 1. I may be wrong but think someone who is at this level can figure out a way to do this for their own env.